src/Haskoin/Address.hs

```diff
@@ -5,16 +5,15 @@
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE OverloadedStrings #-}
```

Comment:
How do you feel about adding `LANGUAGE Safe` or `LANGUAGE Trustworthy` where we can?

Reply:
Think it's great, but not needed for this PR. Can we cut a separate ticket for it just to keep things clean?

Reply:
https://github.com/haskell-bitcoin/bitcoin/issues/14